english/dream/fly/asia/asia.md

```diff
@@ -1,5 +1,69 @@
 You always wanted to visit some exotic countries,
-so you head of to asia.
+so you head off to asia.
 
 (Also you somehow like curries and other take-away foods,
 so why not to grab something authentic?)
+
+Yes, that seems like a good idea. You board the plane for Asia.
+
+Stopping first in India for some of that authentic curry and to play with those
+monkeys that seem to be ubiquitous. 
+
+You are surrounded by vividly dressed people of all kinds. You past many
+explorers like your self, who stand with uninhibited awe and marvel at the wonder all 
+around.
+
+What exotic adventures await?
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
It looks like there's quite a few blank lines at the end of this file! Do you mind removing them?